lib/player.rb

```diff
@@ -0,0 +1,30 @@
+class Player
+  attr_reader :name
+  attr_reader :move
+
+  def initialize(name, robot)
+    @name = name
+    @robot = robot
```

Comment:
You should change @robot to a real boolean, rather than checking "true"-string or nil.
@robot = robot == "true"